app/src/components/common/icons/IconClose.tsx

```diff
@@ -38,7 +39,7 @@ export const IconClose = (props: Props) => {
       <path
         className="path"
         d="M19 6.41L17.59 5L12 10.59L6.41 5L5 6.41L10.59 12L5 17.59L6.41 19L12 13.41L17.59 19L19 17.59L13.41 12L19 6.41Z"
-        fill={hoverEffect ? '#DCDFF2' : color}
```

Comment:
Why was this changed?

Reply:
Because the hover effect is set higher up on the file, line 7-11

![Screenshot 2021-07-01 at 10 07 13](https://user-images.githubusercontent.com/57708955/124089351-1cc96200-da54-11eb-8fc5-efb130cf7b17.png)

Reply:
those two are not the same properties one is for path and other is for fill they affect 2 different parts of svg. Try changing them to different colors to see where they apply....

Reply:
as we discussed on keybase it won't work because there is lot of places where we use hoverEffect and with removing of this it won't apply that color. So we either change it in all places (which is not that dry). So either we leave it as it ....Or there is maybe some other solution that will work even better....